### PR TITLE
Add product database (seeder function)

### PR DESCRIPTION
**Pull request by thottel64**
Seeder function looks to have worked as intended as it was able to seed the products.db file

### CODE REVIEW

File: sqldatabaseproductserver/main.go [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+	//err := os.Remove("products.db")
+	//if err != nil {
+	//	return err
+	//}
+	//sqlStatement := "CREATE TABLE products (ID primary key , Name ,Price); delete from products"
+	//_, err := db.Exec(sqlStatement)
+	//if err != nil {
+	//	fmt.Println("error with sqlstatement")
+	//	return err
+	//}
```
Comment (jboursiquot) [inline sqldatabaseproductserver/main.go]: Don't leave commented-out code behind.

File: sqldatabaseproductserver/main.go [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	transaction, err := db.Begin()
+	if err != nil {
+		fmt.Println("error on line 52")
```
Comment (jboursiquot) [inline sqldatabaseproductserver/main.go]: Remove debug statements.

File: sqldatabaseproductserver/main.go [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+			fmt.Println("error on line 64")
+			return err
+		}
+
+	}
+	err = transaction.Commit()
+	if err != nil {
+		return err
+	}
+	rows, err := db.Query("SELECT ID, Name, Price FROM products WHERE ID <= 5)")
```
Comment (jboursiquot) [inline sqldatabaseproductserver/main.go]: You can use LIMIT instead.

File: sqldatabaseproductserver/main.go [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+	if err != nil {
+		fmt.Println("error with query")
+		return err
+	}
+	fmt.Println(rows)
+	defer rows.Close()
+	return nil
+}
+
+func InitProducts() {
```
Comment (jboursiquot) [inline sqldatabaseproductserver/main.go]: Why not return the []products instead of relying on a global?

**Review by jboursiquot — CHANGES_REQUESTED** [review]

**Review by jboursiquot — CHANGES_REQUESTED** [review]
You still have unaddressed feedback.

File: sqldatabaseproductserver/main.go (hunk 4 above)
Comment (thottel64) [reply sqldatabaseproductserver/main.go]: Can't believe I forgot to address this 🤦

File: SQLDatabaseProductSeeder/main.go [hunk 5]
```diff
@@ -0,0 +1,82 @@
+package main
```
Comment (jboursiquot) [inline SQLDatabaseProductSeeder/main.go]: Just name this folder `seeder` -- less is more.

**Review by jboursiquot — CHANGES_REQUESTED** [review]
Resolve your conflicts.

**Comment by thottel64** [comment]
Conflicts Resolved. Requesting review hopefully for the final time.

**Review by jboursiquot — APPROVED** [review]

**Pull request merged**